01week/bio/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Samantha's Bio Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<div id="wrapper">
+
+	<div id="header">
+      <img src="https://unsplash.com/?photo=21ae7tGpJtE"
+		<div id="nav"><a href="index.html">Home</a><br></br><a href="#">About Me</a><br></br>
+      <a href="#">Contact Me</a<br></br>
+      </div>
+		<div id="bg"></div>
+<h2>
+	<div id="main-content">
+		<div id="left-column">
+			<div id="logo">
+			<h1>Samantha Leal</h1>
+			</div>
+			<div class="box">
+      </h2>
+        		<h1>Who am I?</h1>
+        		<p>This is just a fraction of me!</p>
+            <h3>
+				<ul style="margin:10px 20px 10px 20px;">
+					<li>Hiking the Greenbelt</li>
+					<li>Camping</li>
+					<li>Playing with my dog, Tilley</li>
+					<li>Reading</li>
+				</ul>
+      </h3>
+			</div>
```

Comment:
You have a few tags left open.  I think there are 4 <div> tags left open, as well as your <body> and <html>.